### PR TITLE
BUGFIX: ONE-4454 Column with long content is stretched to show full i…

### PR DESCRIPTION
**Pull request by ivanmjartan**
…ts content

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7269/ : SUCCESS in 2m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3269/ : SUCCESS in 3s

**Comment by ivanmjartan** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1590.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3270/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3180/ : SUCCESS in 19m 21s

**Comment by ivanmjartan** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3813/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3271/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3181/ : SUCCESS in 11m 05s

File: src/components/core/PivotTable.tsx [hunk 1]
```diff
@@ -967,6 +967,29 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
...
+        // for init load this method return page after sort change return last execution rowcount
+        // this value is used in gridOption and remove it unnecessary horizontal scrollbars flickering when
+        // new execution is done after sort change
+        const { execution } = this.state;
+        const { pageSize } = this.props;
+        let rowCount = 0;
+
+        if (
+            execution &&
+            execution.executionResult &&
```
Comment (ivan-nejezchleb) [inline src/components/core/PivotTable.tsx:980]: replace this by this.getExecutionResult()

File: src/components/core/PivotTable.tsx [hunk 2]
```diff
@@ -967,6 +967,29 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
     };
 
+    private getInfiniteInitialRowCountRowCount() {
+        // for init load this method return page after sort change return last execution rowcount
```
Comment (ivan-nejezchleb) [inline src/components/core/PivotTable.tsx:971]: reduce this comment to the explanation why this is done - scrollbar removal. What is does is readable from code

File: src/components/core/PivotTable.tsx [hunk 3]
```diff
@@ -997,6 +1020,7 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
             );
         }
 
+        const rowCount = this.getInfiniteInitialRowCountRowCount();
```
Comment (ivan-nejezchleb) [inline src/components/core/PivotTable.tsx:1023]: some UT would be nice

Comment (ivanmjartan) [reply src/components/core/PivotTable.tsx:1023]: Done

File: src/components/core/PivotTable.tsx (hunk 2 above)
Comment (ivanmjartan) [reply src/components/core/PivotTable.tsx:971]: Done

File: src/components/core/PivotTable.tsx (hunk 1 above)
Comment (ivanmjartan) [reply src/components/core/PivotTable.tsx:980]: Done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7271/ : SUCCESS in 2m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3274/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3815/ : SUCCESS in 13m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/3/ : SUCCESS in 20s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7274/ : SUCCESS in 5m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1778/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2611/ : SUCCESS in 17m 21s

**Pull request merged**